app/helpers/timelines_helper.rb

```diff
@@ -182,4 +182,31 @@ def push_timeline_options(timeline, target = gon)
   def timeline_options
     OpenStruct.new @timeline.options
   end
+
+  def new_timeline_link(project, &block)
+    link_to({ controller: '/timelines', action: 'new', project_id: project },
+            title: l('timelines.new_timeline'),
+            class: 'button -alt-highlight', &block)
```

Comment:
true, true. reads better. 

Altough, I will have to refactor that `link_to` crap at some point anyway - i already tried using the path_helpers where applicable, but some routes are build... strangely.